samples/swtpm-localca.in

```diff
@@ -1,10 +1,5 @@
 #!/usr/bin/env sh
 
-swtpm_localca_exe="$(dirname "$0")/swtpm_localca"
-if ! [ -x "$swtpm_localca_exe" ]; then
-  swtpm_localca_exe=@BINDIR@/swtpm_localca
-fi
-
-$swtpm_localca_exe "$@"
```

Comment:
This logic is unnecessary now that tests call `swtpm_localca` directly at the location where it was built, as all tests do with all the other compiled programs. The wrapper script is now only used post-installation, and no longer by tests.